Python/shapeworks/ShapeworksPython.cpp

```diff
@@ -593,6 +594,7 @@ PYBIND11_MODULE(shapeworks, m)
   .def("centerOfMass",          &Mesh::centerOfMass, "center of mass of mesh")
   .def("numPoints",             &Mesh::numPoints, "number of points")
   .def("numFaces",              &Mesh::numFaces, "number of faces")
+  .def("getPoint",              &Mesh::getPoint, "value at given point", "p"_a)
```

Comment:
Its not really clear what this means. "return (x,y,z) coordinates of the vertex at a given index" perhaps?

Reply:
Updated documentation